### DISCUSSION TITLE
No recent versions of datatables library available

### DISCUSSION THREAD

**Discussion by DmitriIV100** [cdnjs/cdnjs#14167] [General] [answered]
Hello all

I would like to add datatables library (https://cdnjs.com/libraries/datatables) to our ASP.NET Core project via CDNJS, however the latest version available is 1.10.21

This version was released back in May 2020 and currently is 12 releases behind the latest one. In contrast, all other libraries we use in the project (bootstrap, jquery, font-awesome, etc) have their latest versions available. Does this mean that this library is no longer being updated on CDNJS?

Thank you

**Comment by MattIPv4** [comment, marked as answer]
👋 Datatables changed how they were distributed, see https://github.com/cdnjs/packages/issues/1350

I suspect you'll want the newer [datatables.net libraries](https://cdnjs.com/libraries/datatables.net/1.12.1) on cdnjs.

**Reply by DmitriIV100** [reply]
Thank you for the response. Looks like, based on this new packaging, the libraries I need are datatables.net and datatables.net-bs5.

Btw, I noticed that while the latest version on https://datatables.net/ is 1.12.1, the latest version on CDNJS is 2.1.1 with the comment inside the javascript files stating that it is actually 1.10.13.

**Reply by MattIPv4** [reply]
Yes, I suspect they accidentally distributed that version incorrectly. Unfortunately, we cannot remove versions like that as assets on cdnjs are considered immutable.
